examples/decoding/plot_decoding_csp_eeg.py

```diff
@@ -57,6 +57,8 @@
 
 # strip channel names
 raw.info['ch_names'] = [chn.strip('.') for chn in raw.info['ch_names']]
+for ch in raw.info['chs']:
+    ch['ch_name'] = ch['ch_name'].strip('.')
```

Comment:
I haven't looked at that function at all, but it sounds reasonable -- I'll see if I can get it to work